README.md

```diff
@@ -115,146 +60,67 @@ pub fn render() {
 }
 ```
 
-## Building and running
-
-To build your app, run `cargo make build`, and to host on a dev server, run `cargo make serve`.
-
-For a more robust starting setup, check out Martin Kavik's [seed-quickstart-webpack repo](https://github.com/seed-rs/seed-quickstart-webpack).
-
-## Running included examples
-
-To run an example located in the [examples folder](https://github.com/seed-rs/seed/tree/master/examples),
-run `cargo make start example_name`, where you replace `example_name` with the example name. Eg:
-`cargo make start counter`.
-
-Some examples also require to run API server in another terminal window - `cargo make start_server example_name`.
-
-When server(s) are running, open [127.0.0.1:8000](http://127.0.0.1:8000) in your browser.
-
-## Resources
+# Examples
```

Comment:
Maybe we should swap`Examples` and `Quickstart`- you don't want to jump to Quickstart if you don't have any idea how to write apps in Seed. And you don't want to invest your time to learn Quickstart if you are not really convinced by examples that you want to use Seed.

Reply:
The idea was to have the excerpt on the readme be the same code as the quickstart, but I think it does make sense to just put examples first.